### PR TITLE
Add 2 new rules

### PR DESCRIPTION
**Pull request by erik-beus**
- Add a rule for unknown url parameters
- Add/fix the `noReferenceProjectActionRule`

Closes https://github.com/toddledev/toddle-internal/issues/1970

### CODE REVIEW

**Comment by erik-beus** [comment]
> You still have to add the rule to the `problems.worker.ts` file

Lol thank you 😅

File: packages/search/src/rules/noReferenceProjectActionRule.ts [hunk 1]
```diff
@@ -1,88 +1,47 @@
+import { CustomActionModel } from '@toddledev/core/dist/component/component.types'
 import { ToddleComponent } from '@toddledev/core/dist/component/ToddleComponent'
-import { Formula } from '@toddledev/core/dist/formula/formula'
-import { isToddleFormula } from '@toddledev/core/dist/formula/formulaTypes'
 import type { Rule } from '../types'
 
-export const noReferenceProjectFormulaRule: Rule<void> = {
```
Comment (jacob-kofoed) [inline packages/search/src/rules/noReferenceProjectActionRule.ts:6]: What happened to the `noReferenceProjectFormulaRule` 🤔 did you duplicate or change the rule?

Comment (erik-beus) [reply packages/search/src/rules/noReferenceProjectActionRule.ts:6]: The `noReferenceProjectActionRule` was a copy of the `noReferenceProjectFormulaRule` 🤷‍♂️ So it appears we just never finished the rule for unused actions...

File: packages/search/src/rules/unknownUrlParameterRule.test.ts [hunk 2]
```diff
@@ -0,0 +1,99 @@
+import { searchProject } from '../searchProject'
+import { unknownUrlParameterRule } from './unknownUrlParameterRule'
+
+describe('unknownUrlParameterRule', () => {
+  test('should report reading unknown variables', () => {
```
[the remark this answers is not in the document]
Comment (erik-beus) [reply packages/search/src/rules/unknownUrlParameterRule.test.ts:5]: 🤦‍♂️ I missed this one. I'll fix in a separate commit

**Pull request merged**